nixos/modules/services/system/nscd.nix

```diff
@@ -56,6 +56,13 @@ in
   config = mkIf cfg.enable {
     environment.etc."nscd.conf".text = cfg.config;
 
+    users.users.nscd = {
+      isSystemUser = true;
+      group = "nscd";
```

Comment:
Should this be configurable? Maybe internal that we don't hardcode the user.

Reply:
done.